images/scripts/virt-install-rhel4edge

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#
+# Copyright (C) 2015 Red Hat Inc.
+# Author: <kashyap@redhat.com>
+# Further contributions: <pjp@fedoraproject.org>
+# Adapted: <marius.vollmer@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+# Adapted from create-guest-qcow2.  We don't use that script as is
+# since it is, to quote Mr Parker, "2% code and 98% config stuff".
+
+set -ex
+
+if [ "$#" -lt 3 ]; then
+    echo >&2 "Usage: virt-install-rhel4edge IMAGE ARCH URL"
+    exit 1
+fi
+
+out=$1
+arch=$2
+url=$3
+
+base=$(dirname $0)
+ks=fedora.ks
+
+if [ -e "$NODE_EXTRA_CA_CERTS" ]; then
+    crt="$(cat "$NODE_EXTRA_CA_CERTS")"
+    crt_command="echo '$crt' > /root/node_extra_ca_certs"
+fi
```

Comment:
I don't think we'll need that on the image, we are not going to run `npm` on it.